unittests/test_edienergyscraper.py

```diff
@@ -876,7 +876,7 @@ async def test_mirroring(self, mocker, tmpdir_factory, datafiles, caplog):
         ],
     )
     def test_get_edifact_format(
-        self, input_files: list[str], expected_results: list[(EdifactFormatVersion, Optional[EdifactFormat])]
```

Comment:
@hf-sheese: das problem war glaube ich hier entweder das `list[(Foo,Bar)]` oder dass es lowercase `tuple` in python 3.8 noch nicht gab:
https://docs.python.org/3.8/library/typing.html
https://docs.python.org/3.9/library/typing.html
<-- man sieht, dass auch in der doku sich das casing von list[...] und tuple[...] zwichen 3.8 und 3.9 ändert.